packages/client/src/components/UserProfileForm/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { Form, Formik, FormikValues } from 'formik'
+import { TUserData } from '../../api/types'
+import { validate } from '../../utils/validator'
+import { UserProfileFormInput } from '../UserProfileFormInput'
+import { UserProfileSaveButton } from '../UserProfileSaveButton'
+import style from './index.module.scss'
+
+type TUserProfileForm = {
+  isDisable: boolean
+  userData: TUserData
+  onsubmitCallback: (values: FormikValues) => void
+  className?: string
+}
+
+export const UserProfileForm = ({
+  userData,
+  isDisable,
+  onsubmitCallback,
+  className = '',
+}: TUserProfileForm) => {
+  return (
+    <div className={`${style.user_form_wrapper} ${className}`}>
+      <Formik
+        enableReinitialize={true}
+        initialValues={userData}
+        validate={validate}
+        onSubmit={onsubmitCallback}>
+        {({ values }) => (
+          <Form className={style.user_form}>
+            <UserProfileFormInput
+              id="email"
+              type="text"
+              name="email"
+              value={values['email']}
+              labelText="Почта"
+              isDisabled={isDisable}
```

Comment:
не зря называл isDisable**d** :)
```suggestion
              isDisabled={isDisabled}
```